FinniversKit/Sources/Recycling/GridViews/Markets/CompactView/CompactMarketsViewCell.swift

```diff
@@ -0,0 +1,158 @@
+import UIKit
+
+public class CompactMarketsViewCell: UICollectionViewCell {
```

Comment:
Maybe the cell can be internal in this package, since it won't be used outside of it